src/backend/src/controllers/slack.controllers.ts

```diff
@@ -0,0 +1,16 @@
+import { getWorkspaceId } from '../integrations/slack';
+import OrganizationsService from '../services/organizations.services';
+import slackServices from '../services/slack.services';
+
+export default class SlackController {
+  static async processMessageEvent(event: any) {
+    try {
+      const organizations = await OrganizationsService.getAllOrganizations();
+      const nerSlackWorkspaceId = await getWorkspaceId();
+      const orgId = organizations.find((org) => org.slackWorkspaceId === nerSlackWorkspaceId)?.organizationId;
```

Comment:
Small change: I think we should find the organization, and then check if the org exists. The after pass in the organization.organizationId into the function